src/handlers/InstanceHandler.ts

```diff
@@ -4,99 +4,194 @@ import MetagameEventEvent from './census/events/MetagameEventEvent';
 import {jsonLogOutput} from '../utils/json';
 import {getLogger} from '../logger';
 import ApplicationException from '../exceptions/ApplicationException';
-import {InstanceSchemaInterface} from '../models/InstanceModel';
 import {MetagameEventState} from '../constants/metagameEventState';
-import {instanceId} from '../utils/instance';
 import MongooseModelFactory from '../factories/MongooseModelFactory';
 import {TYPES} from '../constants/types';
-import ActiveInstanceAuthority from '../authorities/ActiveInstanceAuthority';
+import {World} from '../constants/world';
+import PS2AlertsInstanceInterface from '../interfaces/PS2AlertsInstanceInterface';
+import PS2AlertsMetagameInstance from '../instances/PS2AlertsMetagameInstance';
+import {Zone} from '../constants/zone';
+import {InstanceMetagameSchemaInterface} from '../models/instance/InstanceMetagame';
+import {InstanceCustomWorldZoneSchemaInterface} from '../models/instance/InstanceCustomWorldZone';
+import {Ps2alertsEventState} from '../constants/ps2alertsEventState';
+import _ from 'lodash';
```

Comment:
Destruct lodash.

Reply:
Explain?

Reply:
Instead of loading the whole library only take the functions you need using destructing of an object/import.

Reply:
```suggestion
import {remove} from 'lodash';
```